xwalk_android_tests.gypi

```diff
@@ -194,7 +194,6 @@
           ],
         },
       ],
```

Comment:
Ahh ok. I just verified the output folder and it didn't create those apk's so you are right. :)
I will close this PR for now. Atleast it's good that I didn't commit this patch directly without reviewing. Thanks for reviewing.